.env.example

```diff
@@ -1,6 +0,0 @@
-DB_NAME=
-DB_USER=
-DB_PASSWORD=
-CLOUD_NAME=
-CLOUD_API_KEY=
-CLOUD_API_SECRET=
```

Comment:
can we leave this file in here? These are nice to have for newcomers to see how their .env files should be structured